### PR TITLE
Set accordion placeholders fork

### PR DESCRIPTION
**Pull request by starryeyez024**

### CODE REVIEW

File: elements/pfe-accordion/src/pfe-accordion-header.scss [hunk 1]
```diff
@@ -3,93 +3,99 @@
...
+
 :host {
-  --pfe-expanded--border-left: transparent;
-
-  display: block;
-  background: var(--pfe-expanded--main);
-  color: var(--pfe-expanded--aux);
-  border-left-width: 3px;
-  border-left-style: solid;
-  border-left-color: var(--pfe-expanded--border-left);
```
Comment (starryeyez024) [inline elements/pfe-accordion/src/pfe-accordion-header.scss]: i moved these into a mixin instead

File: elements/pfe-accordion/src/pfe-accordion-header.scss [hunk 2]
```diff
@@ -3,93 +3,99 @@
...
+  }
 }
-
-:host([color="dark"][aria-expanded="true"]),
-:host([on="dark"][aria-expanded="true"]) {
-  @include pfe-trigger-color-dark-expanded;
+:host([on="dark"]),
+:host([color="darker"]),
+:host([color="darkest"]) {
+  @include pfe-trigger-color-darkest;
```
Comment (starryeyez024) [inline elements/pfe-accordion/src/pfe-accordion-header.scss]: added support for theme colors `darker` and `darkest`, `color="dark"` is not supported

File: elements/pfe-sass/mixins/_mixins.scss [hunk 3]
```diff
@@ -176,111 +225,127 @@
...
+}
+
 @mixin pfe-trigger-button($theme: light, $align: left) {
+    --pfe-accordion--main:               transparent;
+    --pfe-accordion--aux:                #{pfe-color(surface--lightest--text)};
+    --pfe-accordion--focus:              #{pfe-color(surface--lightest--link--focus)};
+    --pfe-accordion--border-left:        3px solid transparent;
+    --pfe-accordion--border-right:       1px solid transparent;
+    --pfe-accordion--border-top-bottom:  0px;
+    --pfe-accordion--border-focus-color: #{pfe-color(surface--accent)};;
```
Comment (starryeyez024) [inline elements/pfe-sass/mixins/_mixins.scss]: idk if accent is a desirable color but i wanted the focus state to be visible whether it was expanded or not

File: elements/pfe-sass/mixins/_mixins.scss [hunk 4]
```diff
@@ -176,111 +225,127 @@
...
+
     margin: 0;
     width: 100%;
     height: auto;
-    border: 1px solid transparent;
     font-family: inherit;
-    font-size: #{pfe-var(font-size)};
+    font-size: 16px; //calc(#{pfe-var(font-size)} * 1.1);
```
Comment (starryeyez024) [inline elements/pfe-sass/mixins/_mixins.scss]: this was the variable it was complaining about, still needs fixing

File: elements/pfe-sass/mixins/_mixins.scss [hunk 5]
```diff
@@ -176,111 +225,127 @@
...
 
 @mixin pfe-trigger-color-dark-expanded {
-    --pfe-expanded--main:         #{pfe-color(surface--darker)};
-    --pfe-expanded--aux:          #{pfe-color(surface--darkest--text)};
-    --pfe-expanded--focus:        #{pfe-color(surface--darker--link--focus)};
-    --pfe-expanded--border-left:  #{pfe-color(surface--lightest)};
+    --pfe-accordion--main:         #{pfe-color(surface--darkest)};
+    --pfe-accordion--aux:          #{pfe-color(surface--darkest--text)};
+    --pfe-accordion--focus:        #{pfe-color(surface--darkest--link--focus)};
+    --pfe-accordion--border-left:  solid 3px #{pfe-color(surface--darkest--link)};
```
Comment (chrisdo1) [inline elements/pfe-sass/mixins/_mixins.scss]: @starryeyez024 One small change: right now the header border is blue on a dark background, it should be white.

**Pull request merged**